packages/@yoda/datacollection/index.js

```diff
@@ -0,0 +1,89 @@
+'use strict'
+
+var crypto = require('crypto')
+var https = require('https')
+var logger = require('logger')('datacollection')
+var DEFAULT_HOST = require('@yoda/env')().datacollection.uploadUrl
+// var DEFAULT_HOST = 'das-tc-service-test.rokid.com'
+var DEFAULT_URI = '/das-tracing-collection/tracingUpload'
+var property = require('@yoda/property')
+
+function _createMd5(rokidId, nonce) {
+  return crypto.createHash('md5')
+    .update(rokidId + 'apiVersion1.0.0' + 'nonce' + nonce + rokidId)
+    .digest('hex')
+    .toUpperCase()
+}
+
+/**
+  * Upload buried data
+  * @function upload
+  * @param {object Array} params - One or more detailed buried point data.
```

Comment:
Define a data type like https://github.com/yodaos-project/yodart/blob/master/packages/%40yoda/cloudgw/index.js#L21 then use it here as:

```js
/**
  * @param {someType[]} params
```